lib/ui/channel_buffers.dart

```diff
@@ -130,7 +130,7 @@ class ChannelBuffers {
   }
 
   /// Returns true on overflow.
-  bool push(String/*!*/ channel, ByteData/*!*/ data, PlatformMessageResponseCallback/*!*/ callback) {
+  bool push(String/*!*/ channel, ByteData/*?*/ data, PlatformMessageResponseCallback/*!*/ callback) {
```

Comment:
Changing this because looking at the code, the data simply flows through `_StoredMessage`, and it's up to the framework to interpret the data. Given that most other platform message API allow `ByteData` to be null, I'm making it nullable as well. However, `handleMessage` below is different. `data` cannot be null there because it's used by control channel only, which seems to have a more specific schema. In particular, it never passes `null`.